def funcion_local():
    x = 10  # Variable local
    print(x)

funcion_local()
# print(x) Esto daría un error ya que x es local a la función
